products/android.md

```diff
@@ -14,6 +14,11 @@ eolColumn: Security Support
 sortReleasesBy: 'release'
 releaseLabel: "Android __RELEASE_CYCLE__ '__CODENAME__'"
 releases:
+  - releaseCycle: "12.1"
+    codename: Snow Cone v2 (aka 12L)
```

Comment:
You can also set a custom `releaseLabel` on a per release cycle basis. So the codename could be cleaner.

Reply:
do you think this looks not clean enough ? really i am not good about those cosmetic things so i will be glad to change which one you guys prefer @captn3m0 @BiNZGi

Reply:
It would look the same, but the difference would be in the API response mainly:

- https://deploy-preview-1086--endoflife-date.netlify.app/api/android.json
- https://deploy-preview-1086--endoflife-date.netlify.app/api/android/12.1.json

Having the codename show up as `Snow Cone v2` would be better

Reply:
i am a bit confused just give me exact what should i change this part and i will do like it